kalite/topic_tools/__init__.py

```diff
@@ -165,7 +165,7 @@ def get_exercise_cache(force=False, language=settings.LANGUAGE_CODE):
                 exercise_template = os.path.join(exercise_lang, exercise_file)
 
 
-            with i18n.translate_block(exercise_lang):
+            with i18n.translate_block(language):
```

Comment:
@rtibbles can you see if this change breaks anything? The only one I can think of is since we don't use `select_best_available_language`, we won't be able to fall back to the  toplevel languages when the dialect versions aren't present (i.e. `pt-BR` won't fall back to `pt` when the former is absent, but the latter is present, It always reverts to `en`).